config/locales/sufia.en.yml

```diff
@@ -443,3 +474,8 @@ en:
       admin_set:
         create: 'Save'
         update: 'Save'
+      sufia_permission_template:
```

Comment:
Why are this translation and `sufia_permission_template_access` prefixed with `sufia_`? Because of how nested attrs work?